toolchain/check/eval.cpp

```diff
@@ -768,15 +762,115 @@ static auto ReplaceTypeWithConstantValue(EvalContext& eval_context,
   return IsConstant(*phase);
 }
 
+static auto ResolveSpecificDeclForInst(EvalContext& eval_context,
+                                       const SemIR::Inst& inst) -> void;
+
+// Resolve the specific declaration in the given instruction field. There is an
```

Comment:
This looks like it's only called from line 1870, after the definition; is the forward declaration still needed? i.e.:

```suggestion
// Resolve the specific declaration in the given instruction field. There is an
```

Reply:
It's not needed, removed!